tensorflow/python/keras/layers/local_test.py

```diff
@@ -288,8 +288,24 @@ def test_locallyconnected_implementation(self, width, data_format):
 
       np.random.seed(1)
       tf_test_util.random_seed.set_seed(1)
-      targets = np.random.randint(0, num_classes, (num_samples,))
-
+      
+      # Following code generates sparse targets and converts them 
+      # to one-hot encoded vectors
+    
+      # Create sparse targets eg. [0,1,2]
+      sparse_targets = np.random.randint(0, num_classes, (num_samples,)) 
+      
+      # Convert to one-hot encoding
+      targets =  np.zeros((sparse_targets.size, sparse_targets.max() + 1)) 
+      targets[np.arange(sparse_targets.size), sparse_targets] = 1 
+      
+      # Final targets:
```

Comment:
Please put this comment block after "# Convert to one-hot encoding", and to semantically separate code blocks, just use one empty line. Thanks!

Reply:
The `Final targets :` one?

Reply:
Made requested changes